src/utils/workspace-config/Cargo.toml

```diff
@@ -11,7 +11,7 @@ repository = { workspace = true }
 [features]
 # some crates opt-in static linking, while some opt-in dynamic linking,
 # so they are two features :)
-rw-static-link = ["static-libz-sys", "static-lzma-sys", "static-sasl2-sys"]
+rw-static-link = ["static-libz-sys", "static-lzma-sys", "static-openssl-sys", "static-sasl2-sys"]
```

Comment:
I suggest to add a new flag, and only turn it on in binary release.
So we still enjoy the benefits of fast compile time in CI & local

Reply:
Done https://github.com/risingwavelabs/risingwave/pull/18689/commits/33f4a2ecf15c3e4d424afcc12386e84ed7d3aba2